xwiki-platform-core/pom.xml

```diff
@@ -138,7 +138,18 @@
             -->
             <!-- Difference related to the Security refactoring. Start a new <revapi.differences> entry for other
                  differences -->
-            
+
+            <revapi.differences>
+              <differences>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.class.removed</code>
+                  <old>class org.xwiki.index.migration.R140200001XWIKI19352DataMigration</old>
+                  <justification>R140200001XWIKI19352DataMigration replaced by R140300000XWIKI19614DataMigration because a link reindexing is required by XWIKI-19614, making an additional reindexing for migrations from version lower than 14.2RC1 useless.</justification>
+                  <criticality>highlight</criticality>
```

Comment:
"highlight" is for important breakage, which is not the case here, so it's supposed to be more "documented" AFAIK.

Reply:
If I read https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRevapiIgnores correctly, `documented` in only for unstable code.

>Semantic changes that are potentially breaking in severity(like adding an annotation) but that, after analyzing we consider to not be a breakage.
>
>    Mark the ignore with <criticality>allowed</criticality>

I agree to change to `allowed`.